### PR TITLE
Only allow files that render to HTML to be written as a Page

### PR DESCRIPTION
**Pull request by ivantsepp**
Followup to #301

### CODE REVIEW

File: lib/jekyll-admin/server/page.rb [hunk 1]
```diff
@@ -34,6 +35,24 @@ class Server < Sinatra::Base
 
       private
 
+      def ensure_html_content
+        unless html_content?
```
Comment (benbalter) [inline lib/jekyll-admin/server/page.rb]: I believe this can be better implemented as a guard statement, e.g., `return if html_content?`

**Comment by ivantsepp** [comment]
Thanks @benbalter for the pointer to the better implementation!

**Comment by benbalter** [comment]
Nice work @ivantsepp! 🌊  🐙 😛

**Comment by mertkahyaoglu** [comment]
Thanks again @ivantsepp , and @benbalter for helping out ❤️

**Comment by ivantsepp** [comment]
Thanks for looking and reviewing y'all =)

**Pull request merged**